rpc/block_svc_routines.c

```diff
@@ -4515,7 +4515,7 @@ block_create_v2_1_svc_st(blockCreate2 *blk, struct svc_req *rqstp)
 
 
   if (blk->rb_size) {
-    GB_ASPRINTF(&rbsize, ",%s=%d", GB_RING_BUFFER_STR, blk->rb_size);
+    GB_ASPRINTF(&rbsize, " control='%s=%d'", GB_RING_BUFFER_STR, blk->rb_size);
```

Comment:
BTW, why not just removing the white space before "control=" and then add it in LINE 4037 to be more readble ?

```
4269 blockResponse *
4270 block_create_common(blockCreate *blk, char *rbsize, char *volServer, char *prio_path)
4271 {
....
4307   if (GB_ASPRINTF(&backstore, "%s %s name=%s size=%zu cfgstring=%s@%s%s/%s%s wwn=%s",
4308                   GB_TGCLI_GLFS_PATH, GB_CREATE, blk->block_name, blk->size,
4309                   blk->volume, volServer?volServer:blk->ipaddr, GB_STOREDIR,
4310                   blk->gbid, rbsize ? rbsize: "", blk->gbid) == -1) {
4311     goto out;
4312   }
....
}
```

Will this make sense ?

Reply:
That will add an extra white space every time create command is called without ring-buffer option (which is quite common case).

I just avoid this by pushing the space before ' control='.

Makesense @lxbsz ?

Reply:
Yeah, that's okay for me :-)